src/game/Tactical/UI_Cursors.cc

```diff
@@ -162,6 +162,34 @@ UICursorID GetProperItemCursor(SOLDIERTYPE* const s, GridNo const map_pos, BOOLE
 
 static void DetermineCursorBodyLocation(SOLDIERTYPE*, BOOLEAN fDisplay, BOOLEAN fRecalc);
 
+static UINT8 ChanceToHit(SOLDIERTYPE* pSoldier, GridNo const map_pos)
+{
+	bool const is_throwing_knife = GCM->getItem(pSoldier->inv[HANDPOS].usItem)->getItemClass() == IC_THROWING_KNIFE;
+	SOLDIERTYPE const* const target = gUIFullTarget;
+	UINT8 chance_to_hit, chance_to_clear, chance_on_target;
+
+	if (!target)
+	{
+		chance_to_hit = SoldierToLocationChanceToGetThrough(pSoldier, map_pos, gsInterfaceLevel, pSoldier->bTargetCubeLevel, 0);
+	}
+	else
```

Comment:
does this also work for melee?

Reply:
I will look into adding melee case to the AAC mod. There is CalcChanceHTH for melee and maybe we would have to consider special cases like if the attacker is unseen.